docs/data/date-pickers/faq/faq.md

```diff
@@ -0,0 +1,28 @@
+---
+productId: x-date-pickers
+title: Frequently Asked Questions - Date and Time pickers
+githubLabel: 'component: pickers'
+packageName: '@mui/x-date-pickers'
+---
+
+# Frequently asked questions
+
+<p class="description">Can't find what you are looking for? The FAQ section has answers to some of the most frequent questions and challenges.</p>
+
+If you still have trouble, you can get in touch with through the [support page](/x/introduction/support/).
```

Comment:
```suggestion
If you still have trouble, you can refer to the [support page](/x/introduction/support/).
```
Replacing `our` with `the` to avoid the use of first person 🙈